recipes/python-lzo/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+export CPPFLAGS="-I$PREFIX/include -I$PREFIX/include/lzo $CPPFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+
+$PYTHON setup.py install
```

Comment:
I got `error: option --single-version-externally-managed not recognized`. Please see: https://travis-ci.org/conda-forge/staged-recipes/builds/258113000#L243